torch/_meta_registrations.py

```diff
@@ -344,7 +348,7 @@ def pick_memory_format():
 
     else:
         out_channels = weight.shape[0]
-        if weight.shape[1] != input_tensor.shape[1] / groups:
+        if weight.shape[1] * groups != input_tensor.shape[1]:
```

Comment:
is this what fixed the xfails for conv above? it seems like the exception above is 'couldn't find a symbolic meta func'

Reply:
Yeah I think so. And also the `copy_` meta registration.

Reply:
The error messages might be out of date :P I can regenerate them.